softwarereview_intro.pt.Rmd

```diff
@@ -44,20 +44,20 @@ Além disso, os autores e revisores podem entrar em contato com os editores em p
 
 - O LEIAME do seu pacote apresentará um selo de revisão por pares com um link para o tópico de revisão do software.
 - Seu pacote receberá um selo [`docs.ropensci.org` site de documentos](#rodocsci) que você pode vincular a partir da DESCRIÇÃO.
-- O repositório do seu pacote será transferido para a organização rOpenSci.
+- Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci.
```

Comment:
Making more clear what we mean by transfering to rOpenSci org. 
But this is not in the English version, check how to proceed.
```suggestion
- Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci no GitHub.
```